includes/blocks/downloads/edit.js

```diff
@@ -389,9 +388,9 @@ class DownloadsEdit extends Component {
 		}
 
 		const request = apiFetch( {
-			url: `${url}/?edd-api=products&${ stringify( {
+			url: `${url}/?edd-api=products&${ {
 				...query
-			} ) }`,
+			} }`,
```

Comment:
@LisaCee According to https://github.com/WordPress/gutenberg/blob/master/packages/api-fetch/README.md#usage I think this needs to be more like

```suggestion
apiFetch( {
  url: `${url}/?edd-api=products`,
  data: query,
} )
```